src/components/card/card.style.ts

```diff
@@ -9,17 +10,29 @@ const paddingSizes = {
   large: "0 48px",
 };
 
+type DesignTokensType = keyof typeof DesignTokens;
+export type BoxShadowsType = Extract<DesignTokensType, `boxShadow${string}`>;
+
 export interface StyledCardProps extends MarginProps {
   cardWidth: string;
   interactive: boolean;
   draggable: boolean;
   spacing: CardSpacing;
+  boxShadow?: BoxShadowsType;
+  hoverBoxShadow?: BoxShadowsType;
 }
 const StyledCard = styled.div<StyledCardProps>`
-  ${({ cardWidth, interactive, draggable, spacing }) => css`
+  ${({
+    cardWidth,
+    interactive,
+    draggable,
+    spacing,
+    boxShadow,
+    hoverBoxShadow,
+  }) => css`
     background-color: var(--colorsUtilityYang100);
     border: none;
-    box-shadow: var(--boxShadow050);
+    box-shadow: ${boxShadow ? `var(--${boxShadow})` : `var(--boxShadow050)`}};
```

Comment:
I think it would be better to set default value for `boxShadow` function argument here

Reply:
Updated to have default values. It uses just the variable itself and populates the rest around it i.e. `var(--${boxShadow})`